compiler/rustc_query_system/src/query/plumbing.rs

```diff
@@ -572,17 +575,15 @@ fn incremental_verify_ich<CTX, K, V>(
 }
 
 #[inline(always)]
-fn force_query_with_job<C, CTX>(
+fn force_query_with_job<CTX, K, V>(
     tcx: CTX,
-    key: C::Key,
-    job: JobOwner<'_, CTX, C>,
+    key: K,
+    job_id: QueryJobId<CTX::DepKind>,
     dep_node: DepNode<CTX::DepKind>,
-    query: &QueryVtable<CTX, C::Key, C::Value>,
-) -> (C::Stored, DepNodeIndex)
+    query: &QueryVtable<CTX, K, V>,
+) -> (V, DepNodeIndex)
 where
-    C: QueryCache,
```

Comment:
Glad to see this `C: QueryCache` clause removed ^^
It is annoying when trying to make things `dyn`.